osu.Game/Screens/Play/Player.cs

```diff
@@ -157,14 +157,16 @@ protected override void LoadComplete()
                 PrepareReplay();
         }
 
-        private Replay recordingReplay;
+        private Score recordingScore;
```

Comment:
Judging from the fact that this is null-checked lower down in a frame count check I'd probably either `Score?` or `[CanBeNull]` this.

Reply:
Can't use `Score?` on non-nullable classes, so let's go with the attrib version for now.